src/libraries/Microsoft.Extensions.Hosting/src/Microsoft.Extensions.Hosting.csproj

```diff
@@ -3,9 +3,11 @@
   <PropertyGroup>
     <TargetFrameworks>$(NetCoreAppCurrent);$(NetCoreAppMinimum);netstandard2.1;netstandard2.0;$(NetFrameworkMinimum)</TargetFrameworks>
     <EnableDefaultItems>true</EnableDefaultItems>
-    <PackageDescription>Hosting and startup infrastructures for applications.</PackageDescription>
     <EnableAOTAnalyzer>true</EnableAOTAnalyzer>
     <IsPackable>true</IsPackable>
+    <PackageDescription>Hosting and startup infrastructures for applications.</PackageDescription>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <ServicingVersion>1</ServicingVersion>
```

Comment:
I don't think we need to service this package just for removing this internal attribute. Can we revert this change?

Reply:
Reverted